pkg/converter/convert.go

```diff
@@ -119,10 +120,10 @@ func Convert(o Options) error {
 				exec.Args = append(exec.Args, serveridArg)
 				exec.Args = append(exec.Args, argLoginMethod)
 				exec.Args = append(exec.Args, o.TokenOptions.LoginMethod)
```

Comment:
different login method will result in different arguments

Reply:
trying to accommodate different arguments